examples/correlation-id.yml

```diff
@@ -20,55 +20,97 @@ servers:
-      - openIdConnectWellKnown: []
-
-defaultContentType: application/json
-
+      - $ref: '#/components/securitySchemes/apiKey'
+      - type: oauth2
+        description: Flows to support OAuth 2.0
+        flows:
+          implicit:
+            authorizationUrl: 'https://authserver.example/auth'
+            availableScopes:
+              'streetlights:on': Ability to switch lights on
+              'streetlights:off': Ability to switch lights off
+              'streetlights:dim': Ability to dim the lights
+          password:
+            tokenUrl: 'https://authserver.example/token'
+            availableScopes:
+              'streetlights:on': Ability to switch lights on
+              'streetlights:off': Ability to switch lights off
+              'streetlights:dim': Ability to dim the lights
+          clientCredentials:
+            tokenUrl: 'https://authserver.example/token'
+            availableScopes:
+              'streetlights:on': Ability to switch lights on
+              'streetlights:off': Ability to switch lights off
+              'streetlights:dim': Ability to dim the lights
+          authorizationCode:
+            authorizationUrl: 'https://authserver.example/auth'
+            tokenUrl: 'https://authserver.example/token'
+            refreshUrl: 'https://authserver.example/refresh'
+            availableScopes:
+              'streetlights:on': Ability to switch lights on
+              'streetlights:off': Ability to switch lights off
+              'streetlights:dim': Ability to dim the lights
+        scopes:
+          - 'streetlights:on'
+          - 'streetlights:off'
+          - 'streetlights:dim'
+      - $ref: '#/components/securitySchemes/openIdConnectWellKnown'
 channels:
-  smartylighting/streetlights/1/0/event/{streetlightId}/lighting/measured:
+  'smartylighting/streetlights/1/0/event/{streetlightId}/lighting/measured':
+    address: 'smartylighting/streetlights/1/0/event/{streetlightId}/lighting/measured'
+    messages:
+      receiveLightMeasurement.message:
+        $ref: '#/components/messages/lightMeasured'
     parameters:
       streetlightId:
         $ref: '#/components/parameters/streetlightId'
-    publish:
-      summary: Inform about environmental lighting conditions of a particular streetlight.
-      operationId: receiveLightMeasurement
-      message:
-        $ref: '#/components/messages/lightMeasured'
-
-  smartylighting/streetlights/1/0/action/{streetlightId}/dim:
+  'smartylighting/streetlights/1/0/action/{streetlightId}/dim':
+    address: 'smartylighting/streetlights/1/0/action/{streetlightId}/dim'
+    messages:
+      dimLight.message:
+        $ref: '#/components/messages/dimLight'
     parameters:
       streetlightId:
         $ref: '#/components/parameters/streetlightId'
-    subscribe:
-      operationId: dimLight
-      message:
-        $ref: '#/components/messages/dimLight'
-
+operations:
+  receiveLightMeasurement:
+    action: receive
+    channel:
+      $ref: >-
```

Comment:
I didn't knew this notion `>-` is it to allow multi-lines in that case why not having `#/channels/smartylighting~1streetlights~11~10~1event~1{streetlightId}~1lighting~1measured` in the same line.

Reply:
It's part of the converter logic somewhere 🤔 

Might be because the string is too long 🧐